frontend/src/component/segments/SegmentTable.tsx

```diff
@@ -182,8 +182,34 @@ const COLUMNS = [
         Header: 'Name',
         accessor: 'name',
         width: '60%',
+        Cell: ({
```

Comment:
I'd enhance existing test for a segmenttable or create a new one to test enabled and disabled color display logic

Reply:
can be in a separate PR